src/components/Explore.jsx

```diff
@@ -1,7 +1,84 @@
-import React from 'react';
+import React, {Fragment} from 'react';
+import {Container, Col, Row, Form, Card, Button, CardDeck } from 'react-bootstrap'; 
+
 
 const Explore = () => {
-  return <div>Hello Explore Page</div>;
+  
+  // making fetch request from the server to get data
+    // let response = await fetch('/api/login', {
+    //   method: 'GET',
+    //   headers: {
+    //     'Content-Type': 'application/json',
+    //   },
+    // });
+  
+  // get technologies
+  const techStack = ['TECH1', 'TECH2', 'TECH1', 'TECH2', 'TECH1', 'TECH2', 'Tech 33'];
+  // Generate checkbox component for each technology 
+  const generateTech = techStack.map( e => {
+     return(<Form>
+        <div key='checkbox' className="mb-5 mt-5 ml-5">
+          <Form.Check type='checkbox'>
+            <Form.Check.Input type='checkbox' isValid size='lg'/>
+           <Form.Check.Label> {e} </Form.Check.Label>
+          </Form.Check>
+        </div>
+      </Form>)
+  })
```

Comment:
Will have to modify backend to provide list of tech stacks as well when making GET request to /explore